### PR TITLE
1 add signup modules

### LINKED ISSUE #1: Add: Signup modules
Is your feature request related to a problem? Please describe.
N/A

Describe how the solution should behave
The modules should be inside the components directory and called inside the navbar because they will be accessible from there.

Screenshots of the design/if any
image

Additional context
Use formik to handle validation.

### PR DESCRIPTION
**Pull request by suateneskoc**
- Sign up and sign in modals are created utilizing headless ui transitions and dialog
- Reset password feature implementation is postponed until resolving backend functionality

### CODE REVIEW

**Comment by suateneskoc** [comment]
Screen Shot 2022-04-05 at 14 39 53
Screen Shot 2022-04-05 at 14 40 02

**Comment by suateneskoc** [comment]
Usage:

1. Create a boolean state for open/closed state of each modal
2. Create instances and pass the state and setter function as props
3. Pass setter functions of the other modal to each other to enable switching between modals

File: src/components/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            console.log(values);
+            setSubmitting(false);
+          }, 1000);
+        }}
+      >
+        {({
+          // handleSubmit,
+          // handleChange,
+          // handleBlur,
+          // values,
```
Comment (MuhannedNoman) [inline src/components/SignUp/SignUp.jsx]: Let's remove anything we commented out and not used.

**Review by MuhannedNoman — DISMISSED** [review]
Hey @suateneskoc nice work here, I liked the FormField and how you made it re-usable. 
I just left a small comment about the commented code otherwise looks good.

**Review by MuhannedNoman — APPROVED** [review]
LGTM, let's add the test in a different PR.
Feel free to merge.

**Pull request merged**